Final name and format of end point

popkinj opened the pull request:
There is one end point /form and all templates will be specified by URL argument name=name-of-template

andrewwhitehead commented on fpo-api/README.md:
````diff
@@ -31,9 +46,13 @@ Migrations are triggered automatically when the Django/Python container is deplo
 ## Testing
 ```bash
+curl -X POST \
+  -d '@/home/popkinj/contracts/quartech/justice/test-data.json' \
````
I don't think you want these paths in the readme?

popkinj commented:
Good eye Andrew... and thanks for catching it.
Just updated and added to PR.

---------
Jamie Popkin
Little Earth
250 390 6816
https://littleearth.ca



On Mon, Apr 6, 2020 at 9:34 PM Andrew Whitehead <notifications@github.com>
wrote:

> *@andrewwhitehead* commented on this pull request.
> ------------------------------
>
> In fpo-api/README.md
> <https://github.com/bcgov/family-law-act-app/pull/35#discussion_r404529442>
> :
>
> > @@ -31,9 +46,13 @@ Migrations are triggered automatically when the Django/Python container is deplo
>
>  ## Testing
>  ```bash
> +curl -X POST \
> +  -d '@/home/popkinj/contracts/quartech/justice/test-data.json' \
>
> I don't think you want these paths in the readme?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bcgov/family-law-act-app/pull/35#pullrequestreview-388776950>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AADU6YTOC3H7VR2J536UVDTRLKUO3ANCNFSM4LVJQVXQ>
> .
>
